scripts/schema_to_dataclass.py

```diff
@@ -0,0 +1,166 @@
+#!/bin/env/python
+import sys
+import re
+import json
+from pathlib import Path
+
+map_schema_type_to_python = {
+    "object": "Dict",
+    "array": "List",
+    "integer": "int",
+    "string": "str",
+    "number": "int",
+    "boolean": "bool",
+    "any": "Any",
+}
+
+
+def create_dataclass(name):
+    return dataclass(name)
+
+
+def create_attribute(name, type, required):
+    name = re.sub('([a-z0-9])([A-Z])', r'\1_\2', name).lower()
+
+    try:
+        type = map_schema_type_to_python[type]
+    except KeyError:
+        breakpoint()
+
+    return attribute(name, type, required)
+
+
+class dataclass:
+    def __init__(self, name):
+        self.name = name
+        self.attrs = []
+
+    def add_attr(self, attr):
+        self.attrs.append(attr)
+
+
+    def __str__(self):
+        output = f"@dataclass\nclass {self.name}Payload:\n"
+
+        if len(self.attrs) == 0:
+            return output + "    pass\n"
+
+        optional_attrs = ""
+        for attr in self.attrs:
+            if attr.required:
+                output += str(attr)
+
+            else:
+                optional_attrs += str(attr)
+
+        return output + optional_attrs
+
+class attribute:
+    def __init__(self, name, type, required):
+        self.name = name
+        self.type = type
+        self.required = required
+
+    def __str__(self):
+        name = self.name
+        if not re.match("^[a-zA-Z_]", self.name):
+            name = "_" + self.name
+
+        definition = f"    {name}: {self.type}"
+        if self.required is True:
+            # if self.type == "Dict":
+                # definition += " = field(default_factory=dict)\n"
```

Comment:
why did you comment it out?

Reply:
I think the script can be improved in several ways. This PR isn't about the script, more about the classes it generates. 

I added the script in order not to lose it. For this PR I ignore comments on the script if you don't mind.